source/api_cc/src/DeepPotPT.cc

```diff
@@ -10,6 +10,13 @@
 #include "device.h"
 #include "errors.h"
 
+#ifdef USE_MPI
```

Comment:
This needs to be given in the CMake file. See op
https://github.com/deepmodeling/deepmd-kit/blob/ba8f52e2d9b7d71b36c4b07962868e3dfeda4ddd/source/op/pt/CMakeLists.txt#L34

Reply:
MPI also needs to be linked, if used.